release-notes/major9/09.21.0/valtimo-frontend-libraries.md

```diff
@@ -26,15 +26,26 @@ The following features were added:
   Eg.: `image/*,application/pdf,.psd`
 
 * **New documents tab for Documenten API uploads**
+
   If `uploadProvider` is set to `UploadProvider.DOCUMENTEN_API` in the environment, a new documents tab is shown. This
   tab only allows one file upload at a time. On selecting a file, a modal opens with a form to input metadata for the
   Documenten API. For more information on how to configure Documenten API uploads, refer to [this page](/using-valtimo/upload/upload-to-documenten-api-with-metadata.md).
 
 * **New datepicker component**
+
   Added a new date picker component to `@valtimo/user-interface`: `v-date-picker`. It is based on [flatpickr](https://flatpickr.js.org/).
   If this component is included anywhere in an implementation, add `"node_modules/flatpickr/dist/flatpickr.css"` to the
   `styles` array in `angular.json` to avoid issues.
 
+* **Added new plugin actions to Documenten API and Zaken API plugin**
+
+  New actions have been added to the Documenten API and Zaken API plugins: **Link uploaded document to zaak** and
```

Comment:
Here as well, I think the document being uploaded does not matter at all. Suggestion is to change this to `Link document to zaak`.